### PR TITLE
Initial 4th shot commit

### PR DESCRIPTION
**Pull request by msusicky**
TODO:
- [ ] compute metrics for vaccinated: ```flask compute-metrics vaccinated 2022-07-xx 2022-08-xx```

### CODE REVIEW

**Review by jan-stanek — APPROVED** [review]
LGTM, is it still draft?

**Comment by msusicky** [comment]
Yes, it is still necessary to add reports to have the visibility for 4th shot + we have to discuss how to work with "posilujici davka"

File: app/templates/macros/graphs/ockovani_celkem.html [hunk 1]
```diff
@@ -30,6 +30,15 @@
             hovertemplate: "%{y}"
         }
 
+        const ockovani_4 = {
+            x: [{% for idx, row in data.iterrows() %}"{{ idx }}",{% endfor %}],
+            y: [{% for idx, row in data.iterrows() %}{{ row['ockovani_4'] }},{% endfor %}],
+            mode: "lines",
+            type: "scatter",
+            name: "Očkovaní 4. posilující dávkou",
```
Comment (jan-stanek) [inline app/templates/macros/graphs/ockovani_celkem.html:38]: this label is not consistenst with 3, dose

File: app/templates/macros/graphs/ockovani.html [hunk 2]
```diff
@@ -113,6 +123,16 @@ <h6>Poznámky</h6>
             hovertemplate: "%{y}"
         }
 
+        const rezervace_4 = {
```
Comment (jan-stanek) [inline app/templates/macros/graphs/ockovani.html:126]: reservations for 4th doses doesn't exist

File: app/templates/macros/graphs/ockovani_celkem.html (hunk 1 above)
Comment (jan-stanek) [reply app/templates/macros/graphs/ockovani_celkem.html:38]: 2. posilující or 4.

File: app/templates/okres.html [hunk 3]
```diff
@@ -59,14 +59,12 @@ <h3>Okres: {{ okres.nazev }} </h3>
...
-                            {{ metrika('Čekající na 3. dávku', metriky.rezervace_cekajici_3, delta_day=metriky.rezervace_cekajici_3_zmena_den, delta_week=metriky.rezervace_cekajici_3_zmena_tyden, warning='Počet registrovaných s rezervovaným termínem na 3.&nbsp;dávku očkování.<br>V současnosti jsou k dispozici data jen pro následující měsíc!') }}
-                            {{ metrika('Čekající na 4. dávku', metriky.rezervace_cekajici_3, delta_day=metriky.rezervace_cekajici_4_zmena_den, delta_week=metriky.rezervace_cekajici_4_zmena_tyden, warning='Počet registrovaných s rezervovaným termínem na 4.&nbsp;dávku očkování.<br>V současnosti jsou k dispozici data jen pro následující měsíc!') }}
+                            {{ metrika('Čekající na posilující dávku', metriky.rezervace_cekajici_3, delta_day=metriky.rezervace_cekajici_3_zmena_den, delta_week=metriky.rezervace_cekajici_3_zmena_tyden, warning='Počet registrovaných s rezervovaným termínem na 3.&nbsp;dávku očkování.<br>V současnosti jsou k dispozici data jen pro následující měsíc!') }}
```
Comment (jan-stanek) [inline app/templates/okres.html:62]: posilujici

File: app/templates/macros/tables/stav_ockovani.html [hunk 4]
```diff
@@ -9,13 +9,13 @@
             <th scope="col" class="text-center" style="width: 15%">
                 Ve frontě
                 <i class="fas fa-info-circle" data-toggle="tooltip" data-placement="auto"
-                   title="Počet čekajících ve frontě (bez vybraného termínu) na očkování 1. nebo 3. dávkou.">
+                   title="Počet čekajících ve frontě (bez vybraného termínu) na očkování 1., nebo posilující dávkou.">
```
Comment (jan-stanek) [inline app/templates/macros/tables/stav_ockovani.html:12]: without `,`

File: app/templates/okres.html [hunk 5]
```diff
@@ -59,12 +59,12 @@ <h3>Okres: {{ okres.nazev }} </h3>
...
                             {#
                             {{ metrika('Dnešní kapacita 2. dávky', metriky.rezervace_kapacita_2, delta_day=metriky.rezervace_kapacita_2_zmena_den, delta_week=metriky.rezervace_kapacita_2_zmena_tyden, color='std', info='Dnešní kapacita pro očkování 2.&nbsp;dávkou.<br>Získáno z maximální kapacity uvedené v rezervačním systému.') }}
                             #}
-                            {{ metrika('Dnešní kapacita 3. dávky', metriky.rezervace_kapacita_3, delta_day=metriky.rezervace_kapacita_3_zmena_den, delta_week=metriky.rezervace_kapacita_3_zmena_tyden, color='std', info='Dnešní kapacita pro očkování 3.&nbsp;dávkou.<br>Získáno z maximální kapacity uvedené v rezervačním systému.') }}
+                            {{ metrika('Dnešní kapacita posilující dávky', metriky.rezervace_kapacita_3, delta_day=metriky.rezervace_kapacita_3_zmena_den, delta_week=metriky.rezervace_kapacita_3_zmena_tyden, color='std', info='Dnešní kapacita pro očkování 3.&nbsp;dávkou.<br>Získáno z maximální kapacity uvedené v rezervačním systému.') }}
```
Comment (jan-stanek) [inline app/templates/okres.html:67]: posilující ještě tady

**Pull request merged**